### ISSUE TITLE
Problem with docker build

### ISSUE THREAD

**Issue by benzhi2011** [dotnet/docs#5416]
Step 7/17 : COPY TestDocker/TestDocker.csproj TestDocker/
docker : COPY failed: CreateFile \\?\C:\ProgramData\Docker\tmp\docker-builder208312035\TestDocker\TestDocker.csproj: The system cannot find the path specified.
I got about error when trying to run docker build -t aspnetapp .
C:\ProgramData\Docker\tmp exists, but docker-builder208312035 is not created. 
Copy-Item "C:\tfs\TestDocker\TestDocker\TestDocker.csproj" -Destination "C:\ProgramData\Docker\tmp"  *****works*******
Copy-Item "C:\tfs\TestDocker\TestDocker\TestDocker.csproj" -Destination "C:\ProgramData\Docker\tmp\docker-builder208312035\TestDocker"  *****Copy-Item : Could not find a part of the path 'C:\ProgramData\Docker\tmp\docker-builder208312035\TestDocker' *******

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9731aae1-505b-abb1-0d4b-d6a83ec735e1
* Version Independent ID: 760790d5-0e81-ea6c-19ea-f226d08ddf1f
* Content: [Building .NET Core Docker Images](https://docs.microsoft.com/en-us/dotnet/core/docker/building-net-docker-images?view=aspnetcore-2.0)
* Content Source: [docs/core/docker/building-net-docker-images.md](https://github.com/dotnet/docs/blob/master/docs/core/docker/building-net-docker-images.md)
* Service: **dotnet-core**
* GitHub Login: @JRAlexander
* Microsoft Alias: **johalex**

**Comment by benzhi2011** [comment]
I have posted one issue [here](https://stackoverflow.com/questions/50490550/cannot-run-dotnet-core-2-1-project-on-docker/50634071#50634071).
Someone said might be problem with the docker version, so I remove the latest version and installed Version 18.03.1-ce-win65 (17513), docker-compose 1.21.1.
If I run docker build -t aspnetapp . from Powershell(as ADMIN) , I got up to step 7/18
Removing intermediate container 17c29381e2f4
 ---> 25f047a30977
Step 7/18 : COPY TestDocker/TestDocker.csproj TestDocker/
COPY failed: stat /var/lib/docker/tmp/docker-builder113779152/TestDocker/TestDocker.csproj: no such file or directory.
I cannot find this DIR and I am using a domain manged win 10 PC version 1703.
